tweets-service/service.yml

```diff
@@ -27,3 +27,10 @@ messages:
 dependencies:
   mongo:
     prod: 'ds143608.mlab.com:43608'
+
+
+production:
+  aws-task-definition:
```

Comment:
Addressed in commit that closes this PR https://github.com/Originate/space-tweet/commit/ce83c3d0430bacf34e8e1423b5b778b9b97fe964